### PR TITLE
Vectors module

### PR DESCRIPTION
**Pull request by shaiTheKimhi**

### CODE REVIEW

File: Modules/Vector/Vector3D.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#/usr/bin/python3
+from math import sqrt
+class vector3D:
+    def __init__(self,x:float=0,y:float=0,z:float=0):
+        self.x = x
+        self.y = y
+        self.z = z
+    def distanceTo(self,vector)->int:
```
Comment (yotam180) [inline Modules/Vector/Vector3D.py]: can you rename this to `distance_to`? Or just `distance` if you don't like the underscore.

**Review by yotam180 — CHANGES_REQUESTED** [review]
And can you add some spacing between operators?
Thanks.

File: Modules/Vector/Vector2D.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#/usr/bin/python3
+from math import sqrt
+class vector2D:
+    def __init__(self,x:float=0,y:float=0):
+        self.x = x
+        self.y = y
+    def distanceTo(self,vector):
```
Comment (yotam180) [inline Modules/Vector/Vector2D.py]: Same request as in the review. Please? :pray:

**Review by avisalmon — COMMENTED** [review]
This is a test review to check if all repository-owners can make the reviews.

File: Modules/Vector/check.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import vector2D
+def func(x:int,v:str)->int:
+	print (v*x)
+	return 3
+#print(func(5,"hello"))
+v = vector2D.vector2D(3,4)
+v2 = vector2D.vector2D(5,100)
+print(v.length()+v2.length())
+v = v+v2
+print(v.length())
```
Comment (ron-popov) [inline Modules/Vector/check.py]: Shouldn't this be in the Tests folder ?

Comment (shaiTheKimhi) [reply Modules/Vector/check.py]: This is a temporary file.
It should be deleted.

**Review by yotam180 — APPROVED** [review]
Now it looks good. Any comments, @ron-popov?

**Comment by shaiTheKimhi** [comment]
@yotam180  your addition of type specification caused an exception on Modules/Vector/Vector3D.py
and Modules/Vector/Vector2D.py

**Pull request merged**